helm-charts/strimzi-kafka-operator/values.yaml

```diff
@@ -97,6 +97,16 @@ kafkaMirrorMaker2:
     repository: strimzi
     name: kafka
     tagPrefix: latest
+cruiseControl:
```

Comment:
I do not see any updated to the Helm Chart REAMDE.md file. Can you update it as well please?